frontend/src/host_orchestrator/orchestrator/controller.go

```diff
@@ -154,12 +155,13 @@ func (h *fetchArtifactsHandler) Handle(r *http.Request) (interface{}, error) {
 		return nil, operator.NewBadRequestError("Malformed JSON in request", err)
 	}
 	creds := r.Header.Get(HeaderBuildAPICreds)
+	projectID := r.Header.Get(HeaderUserProject)
```

Comment:
in the HO context, let's use the term `credsUserProjectID`, `projectID` is confusing as it could represent the GCP project the HO is hosted in as well.